src/main/java/spireMapOverhaul/zones/thieveshideout/ThievesHideoutZone.java

```diff
@@ -0,0 +1,137 @@
+    public static final String THIEF_KING = SpireAnniversary6Mod.makeID("THIEF_KING");
+
+    public ThievesHideoutZone() {
+        super(ID, Icons.MONSTER);
+        this.width = 2;
+        this.height = 5;
+    }
+
+    @Override
+    public AbstractZone copy() {
+        return new ThievesHideoutZone();
+    }
+
+    @Override
+    public Color getColor() {
+        return new Color(0.2f,0.6f, 0.7f, 1f);
+    }
+
+    @Override
+    public boolean canSpawn() {
+        return this.isAct(2);
+    }
+
+    @Override
+    protected boolean canIncludeEarlyRows() {
+        //Since this zone always has a dangerous event node (around super-elite difficulty), we don't want it to show up in the rows of the act that never have elites
+        //It also has normal fights that are tuned as hard pool fights, so that's another reason to prevent early spawns
+        return false;
+    }
+
+    @Override
+    public void manualRoomPlacement(Random rng) {
+        //We put the special node with the Thief King fight in the last row of the zone (that isn't a protected row)
+        List<MapRoomNode> possibleNodes = new ArrayList<>();
+        for (MapRoomNode node : this.nodes) {
+            if (node.getRoom() == null && !this.isProtectedRow(node.y)) {
+                possibleNodes.add(node);
+            }
+        }
+        possibleNodes.stream()
+                .map(n -> n.y).max(Comparator.comparingInt(y -> y))
+                .ifPresent(maxY -> possibleNodes.removeIf(n -> n.y != maxY));
+
+        MapRoomNode node = possibleNodes.get(rng.random(possibleNodes.size() - 1));
+        node.setRoom(new ForcedEventRoom(ThiefKingEvent::new));
+    }
+
+    @Override
+    public void replaceRooms(Random rng) {
+        //No elites in the zone, just the Thief King fight (which awards as much as two elites)
+        for (MapRoomNode node : this.nodes) {
+            if (node.room instanceof MonsterRoomElite) {
+                node.room = new MonsterRoom();
+            }
+        }
+    }
+
+    @Override
+    public List<ZoneEncounter> getNormalEncounters() {
+        return Arrays.asList(
+            new ZoneEncounter(TWO_MUGGERS, 2, () -> new MonsterGroup(
+                new AbstractMonster[] {
+                    new Mugger(-200.0F, 15.0F),
+                    new Mugger(80.0F, 0.0F)
+                })),
+            new ZoneEncounter(LOOTER_AND_RED_SLAVER, 2, () -> new MonsterGroup(
+                new AbstractMonster[] {
+                    new Looter(-270.0F, 15.0F),
+                    new SlaverRed(130.0F, 0.0F)
+                })),
+            new ZoneEncounter(MUGGER_AND_BLUE_SLAVER, 2, () -> new MonsterGroup(
+                new AbstractMonster[] {
+                    new Looter(-270.0F, 15.0F),
+                    new SlaverRed(130.0F, 0.0F)
+                })),
+            new ZoneEncounter(THREE_LOOTERS_WEAK, 2, () -> new MonsterGroup(
```

Comment:
All the other fights seemed reasonably balanced for the hard pool purely with the mix of base game enemies I made, but I wanted a three Looters and that would normally be 33 damage on turn 1. That's a bit crazy for act 2.

Giving them all -2 strength (implemented in the `WeakLooter` class) makes it into 27 damage. That's high, but it's line with other dangerous act 2 fights in the hard pool. In particular, it's the same as Shelled Parasite and Fungi Beast on turn 1.

(All numbers are A20 values.)